e2e-test/.env.example

```diff
@@ -0,0 +1 @@
+CYPRESS_ROOT_SECRET=root-secret
```

Comment:
I think we can use the ROOT_SECRET itself. I cannot think of a scenario where the rootsecret of cypress is a different one than the one in the api or elsewhere

Reply:
@Stezido I'm not sure if that is possible since cypress is consuming only env variables starting with `CYPRESS_`

Reply:
https://docs.cypress.io/guides/guides/environment-variables.html#Option-3-CYPRESS

Reply:
true, thanks for the information link